rueidis.go

```diff
@@ -201,6 +204,14 @@ type SentinelOption struct {
 	ClientName string
 }
 
+// ClusterTopologyRefreshmentOption is the options for the cluster topology refreshment.
+// Cluster Refresh happens only when the cluster topology is not up-to-date.
+type ClusterTopologyRefreshmentOption struct {
```

Comment:
ClusterTopologyRefreshmentOption is too limited. How about just ClusterOption?

Reply:
Ok, [09a9403](https://github.com/redis/rueidis/pull/638/commits/09a9403301ee2aecab963f684a7c499f93a79039)